doctrine/phpcr-dbal-symfony-pack/1.0/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "bundles": {
+        "Doctrine\\Bundle\\DoctrineBundle\\DoctrineBundle": ["all"],
```

Comment:
this is wrong. You should not register bundles from other packages. The recipes of these other packages will already handle it (and this might cause issues with double-registration, or broken unregistration on removal)

Reply:
you are right